src/main/java/frc/robot/RobotContainer.java

```diff
@@ -470,65 +481,188 @@ private void configureDriverBindings(PatriBoxController controller) {
 
         controller.povRight()
-            .onTrue(ampper.toggleSpeed());
-        
+                .onTrue(ampper.toggleSpeed());
+
         controller.povDown()
-            .onTrue(pieceControl.elevatorToBottom());
+                .onTrue(pieceControl.elevatorToBottom());
 
         controller.leftBumper()
-            .whileTrue(pieceControl.intakeUntilNote())
-            .negate().and(driver.leftBumper().negate())
+                .whileTrue(pieceControl.intakeUntilNote())
+                .negate().and(driver.leftBumper().negate())
                 .onTrue(pieceControl.stopIntakeAndIndexer());
 
         controller.rightBumper()
-            .onTrue(pieceControl.ejectNote())
-            .onFalse(pieceControl.stopEjecting());
+                .onTrue(pieceControl.ejectNote())
+                .onFalse(pieceControl.stopEjecting());
 
         controller.rightTrigger()
-            .onTrue(pieceControl.noteToTarget(swerve::getPose, swerve::getRobotRelativeVelocity, swerve::atHDCAngle, () -> controller.getLeftBumper())
-            .andThen(limelight3.setLEDState(() -> false))); 
+                .onTrue(pieceControl
+                        .noteToTarget(swerve::getPose, swerve::getRobotRelativeVelocity, swerve::atHDCAngle,
+                                () -> controller.getLeftBumper())
+                        .alongWith(driver.setRumble(() -> 0.5, 0.3))
+                        .andThen(Commands.runOnce(() -> RobotContainer.hasPiece = false)));
 
         controller.x()
-            .onTrue(pieceControl.setShooterModeCommand(true));
+                .onTrue(pieceControl.setShooterModeCommand(true));
 
         controller.b()
-            .onTrue(pieceControl.setShooterModeCommand(false));
-            
+                .onTrue(pieceControl.setShooterModeCommand(false));
+
         controller.a()
-            .onTrue(pieceControl.panicEjectNote())
-            .onFalse(pieceControl.stopPanicEject());
+                .onTrue(pieceControl.panicEjectNote())
+                .onFalse(pieceControl.stopPanicEject());
 
         controller.start().or(controller.back()).or(controller.y())
-            .whileTrue(pieceControl.sourceShooterIntake(controller.start().or(controller.back()).or(controller.y())))
-            .onFalse(pieceControl.stopIntakeAndIndexer());
+                .whileTrue(
+                        pieceControl.sourceShooterIntake(controller.start().or(controller.back()).or(controller.y())))
+                .onFalse(pieceControl.stopIntakeAndIndexer());
 
         controller.rightStick().and(controller.leftStick())
-            .onTrue(elevator.resetEncoder());
+                .onTrue(elevator.resetEncoder());
 
         controller.rightStick().toggleOnTrue(
-            elevator.overrideCommand(() -> Units.inchesToMeters(controller.getRightY()))
-        );
-        
+                elevator.overrideCommand(() -> Units.inchesToMeters(operator.getRightY())));
+
```

Comment:
Bro just came in, swiped the formatting, and thought he could get away with it 😲
I liked it the way it was before

Reply:
oops